### DISCUSSION TITLE
Refatoração

### DISCUSSION THREAD

**Discussion by daniel-crudo** [pilhacheia/clube-do-livro-dev#17] [Books]
[Refatoração](https://www.amazon.com.br/Refatora%C3%A7%C3%A3o-Aperfei%C3%A7oando-design-c%C3%B3digos-existentes-ebook/dp/B087N8LKYB/ref=sr_1_2?__mk_pt_BR=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=2F3NM91QU0K4C&dchild=1&keywords=refatora%C3%A7%C3%A3o&qid=1620915445&s=digital-text&sprefix=Refatora%C3%A7%C3%A3o%2Cdigital-text%2C287&sr=1-2)

**Comment by lincolixavier** [comment]
Eu adoro esse livro!
